MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -13,25 +13,34 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 final class FeedPresenter {
 	private let feedView: FeedView
 	private let loadingView: FeedLoadingView
+	private let feedErrorView: FeedErrorView
```

Comment:
`feedErrorView` could be named just `errorView` to simplify - the context is clear here so we can avoid repeating "feed" many times (like the `loadingView` is not called `feedLoadingView`).

```suggestion
	private let errorView: FeedErrorView
```

If you follow this suggestion, please, rename it everywhere in this class.